tests/Cargo.toml

```diff
@@ -3,22 +3,26 @@ name = "twilight-book"
 version = "0.1.0"
 authors = ["Twilight Contributors"]
 edition = "2018"
+build = "build.rs"
 
 [dependencies]
 futures = { version = "0.3", default-features = false }
 skeptic = "0.13.5"
 tokio = { version = "1.0", features = ["full"] }
-twilight-gateway = { version = "0.5" }
-twilight-http = { version = "0.5" }
-twilight-model = { version = "0.5" }
-twilight-lavalink = { version = "0.5" }
-twilight-cache-inmemory = { version = "0.5" }
-twilight-command-parser = { version = "0.5" }
-twilight-embed-builder = { version = "0.5" }
-twilight-mention = { version = "0.5" }
-twilight-standby = { version = "0.5" }
-twilight-util = { version = "0.5", features = ["full"] }
-tracing-subscriber = "0.2"
+twilight-gateway = { version = "0.7" }
+twilight-http = { version = "0.7" }
+twilight-model = { version = "0.7" }
+twilight-lavalink = { version = "0.7" }
+twilight-cache-inmemory = { version = "0.7" }
+twilight-command-parser = { version = "0.7" }
+twilight-embed-builder = { version = "0.7" }
+twilight-mention = { version = "0.7" }
+twilight-standby = { version = "0.7" }
+twilight-util = { version = "0.7", features = ["full"] }
+tracing-subscriber = "0.3"
 
 [build-dependencies]
 skeptic = "0.13.5"
+
+[dev-dependencies]
+skeptic = "0.13"
```

Comment:
is this meant to be duplicated?

Reply:
This is how skeptic recommends it upstream, though I can check if it is really needed.

Reply:
Oh, I didn't notice one was build dependencies and one was dev.